_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhdp2.libraries.yml

```diff
@@ -25,11 +25,9 @@ base-theme:
       https://static.redhat.com/libs/redhat/redhat-theme/2/advanced-theme.css: { type: external }
       # NOTE: Provides layout classes for grids and alignments.
       css/pfe-layouts/pfe-layouts.min.css: { minified: true }
-      css/rhd.min.css: { minified: true }
-      css/rhd.microsites.css: { minified: true }
-      css/rhd.legacy.css: { minified: true }
-    theme:
-      css/rhdp2.css: {}
+      rhd-frontend/dist/css/rhd.min.css: { minified: true }
+      rhd-frontend/dist/css/rhd.microsites.css: { minified: true }
```

Comment:
This file is empty, can we remove it here?
https://developer-preview-3486.ext.us-west.dc.preprod.paas.redhat.com/themes/custom/rhdp2/rhd-frontend/dist/css/rhd.microsites.css

Reply:
Yes, removed 👍